lib/azure/armrest/storage/raw_blob_helper.rb

```diff
@@ -0,0 +1,123 @@
+module Azure::Armrest::Storage::RawBlobHelper
```

Comment:
Maybe instead of "RawBlobHelper" we just call it "ManagedStorageHelper" in case we decide to add any other common set of methods.